qa/tasks/cephfs/test_admin.py

```diff
@@ -1610,10 +1610,10 @@ def test_single_path_authorize_on_nonalphanumeric_fsname(self):
         That fs authorize command works on filesystems with names having [_.-]
         characters
         """
-        self.skipTest('this test is broken ATM, see: '
-                      'https://tracker.ceph.com/issues/66077')
-
         self.mount_a.umount_wait(require_clean=True)
+        self.mount_b.umount_wait(require_clean=True)
+        # so that there's no confusion about whether or not we have
```

Comment:
why is explicitly deleting the object required?

Reply:
It's not required. https://tracker.ceph.com/issues/66077#note-9

Reply:
then this hunk is to be removed, yes?

Reply:
We need to unmount before deleting the the Python object.

Reply:
Just unmounting should suffice.